parts/k8s/cloud-init/artifacts/cse_main.sh

```diff
@@ -109,6 +109,9 @@ if [[ "${GPU_NODE}" = true ]]; then
     ensureGPUDrivers
 fi
 {{end}}
+if [[ -n "${PRIVATE_AZURE_REGISTRY_SERVER:-}" ]] && [[ "$CONTAINER_RUNTIME" == "docker" ]]; then
```

Comment:
Oh I see. Does moving it out of the install script make the private login more general?